apps/showcase/src/app/modules/use-http-image-source-pipe-showcase/components/use-http-image-source-pipe-showcase/use-http-image-source-pipe-showcase.component.html

```diff
@@ -0,0 +1,17 @@
+<h1>Use Http Image Source Pipe</h1>
+
+<p>While the requests load, the loading image is displayed</p>
+
+<h2>Success scenario</h2>
+<p>The following image displays a happy perfbuddy when the image is returned</p>
+
+<img width="200px" [src]="'assets/images/success.png' | useHttpImgSrc" />
+
+<br />
+<br />
+<br />
```

Comment:
Can we not use `<br />` for creating a layout?

Reply:
okay, done